qemu/tests/nic_opt.py

```diff
@@ -0,0 +1,149 @@
+import os
+import logging
+import time
+
+from virttest import error_context
+from virttest import utils_net
+from virttest import utils_netperf
+from virttest import utils_misc
+from virttest import utils_test
+from virttest import data_dir
+from avocado.utils import cpu
```

Comment:
import order should be

import build-in modules

import third-party modules

import project modules

Reply:
Thanks, I will replace it.